docs/essentials/sensors.md

```diff
@@ -0,0 +1,540 @@
+            }
+        }
+
+        void Gyroscope_ReadingChanged(object sender, GyroscopeChangedEventArgs e)
+        {
+            GyroscopeData data = e.Reading;
+
+            // Process Angular Velocity X, Y, and Z reported in rad/s
+            Console.WriteLine($"Reading: X: {data.AngularVelocity.X}, Y: {data.AngularVelocity.Y}, Z: {data.AngularVelocity.Z}");
+        }
+    }
+}
+```
+
+### Platform-specific information (Gyroscope)
+
+There isn't any platform-specific information for the gyroscope sensor.
+
+## Magnetometer
+
+The magnetometer sensor indicates the device's orientation relative to Earth's magnetic field.
+
+To start monitoring the magnetometer sensor, call the `Magnetometer.Start` method. .NET MAUI sends magnetometer data changes to your app by raising the `Magnetometer.ReadingChanged` event. The data provided by this event is measured in rad/s (radian per second). Use the `Magnetometer.Stop` method to stop monitoring the sensor. You can detect the monitoring state of the magnetometer with the `Magnetometer.IsMonitoring` property, which will be `true` if the magnetometer was started and is currently being monitored.
+
+All data is returned in $µT$ (microteslas).
+
+```csharp
+using Microsoft.Maui.Essentials;
+using System;
+
+namespace HelloMaui.Sensors
+{
+    public class MagnetometerTest
+    {
+        public void ToggleMagnetometer()
+        {
+            const SensorSpeed speed = SensorSpeed.UI;
+
+            try
+            {
+                if (Magnetometer.IsMonitoring)
+                {
+                    Magnetometer.Stop();
+                    Magnetometer.ReadingChanged -= Magnetometer_ReadingChanged;
+                }
+                else
+                {
+                    Magnetometer.ReadingChanged += Magnetometer_ReadingChanged;
+                    Magnetometer.Start(speed);
+                }
+            }
+            catch (FeatureNotSupportedException fnsEx)
+            {
+                // Feature not supported on device
+            }
+            catch (Exception ex)
+            {
+                // Other error has occurred.
+            }
+        }
+
+        void Magnetometer_ReadingChanged(object sender, MagnetometerChangedEventArgs e)
+        {
+            MagnetometerData data = e.Reading;
+
+            // Process MagneticField X, Y, and Z
+            Console.WriteLine($"Reading: X: {data.MagneticField.X}, Y: {data.MagneticField.Y}, Z: {data.MagneticField.Z}");
+        }
+    }
+}
+```
+
+### Platform-specific information (Magnetometer)
+
+There isn't any platform-specific information for the magnetometer sensor.
+
+## Orientation
+
+The orientation sensor monitors the orientation of a device in 3D space.
+
+> [!NOTE]
+> This sensor isn't used for determining if the device's video display is in portrait or landscape mode, use the `Orientation` property of the `ScreenMetrics` object available from the [`DeviceDisplay`](device-display.md) class.
```

Comment:
Shouldn't the comma be a period, so that a new sentence begins? e.g. This sensor......mode. Instead, use the .... to determine the device's video display orientation.